libraries/localization-utilities/src/TypingsGenerator.ts

```diff
@@ -33,17 +52,25 @@ export interface ITypingsGeneratorOptions extends ITypingsGeneratorBaseOptions {
  */
 export class TypingsGenerator extends StringValuesTypingsGenerator {
   public constructor(options: ITypingsGeneratorOptions) {
-    const { ignoreString, processComment } = options;
+    const {
+      ignoreString,
+      processComment,
+      resxNewlineNormalization,
+      ignoreMissingResxComments,
+      inferDefaultExportInterfaceNameFromFilename
+    } = options;
     super({
       ...options,
       fileExtensions: ['.resx', '.resx.json', '.loc.json', '.resjson'],
-      parseAndGenerateTypings: (fileContents: string, filePath: string, resxFilePath: string) => {
+      parseAndGenerateTypings: (content: string, filePath: string, resxFilePath: string) => {
         const locFileData: ILocalizationFile = parseLocFile({
-          filePath: filePath,
-          content: fileContents,
+          filePath,
+          content,
+          // Explicitly grab this from `this._options` as `this._options.terminal` is initialized later
+          // by the `TypingsGenerator` (from @rushstack/typings-generator) constructor if it isn't provided.
           terminal: this._options.terminal!,
```

Comment:
Can we change this to a protected getter on `TypingsGenerator`? I don't like having the non-null assertion on a value in `this._options`.